printer_zpl2/views/printing_label_zpl2.xml

```diff
@@ -53,15 +53,16 @@
                             </group>
                             <group>
                                 <field name="data" attrs="{'invisible': [('component_type', 'in', ('rectangle', 'circle'))]}"/>
+                                <field name="graphic_image" attrs="{'invisible': [('component_type', '!=', 'graphic')]}"/>
                                 <field name="sublabel_id" attrs="{'invisible': [('component_type', '!=', 'sublabel')]}"/>
                             </group>
                         </group>
                         <notebook colspan="4">
-                            <page string="Format" attrs="{'invisible': [('component_type', '=', ('sublabel'))]}">
+                            <page string="Format" attrs="{'invisible': [('component_type', 'in', ('sublabel'))]}">
```

Comment:
The domain doesn't contain a tuple, but a string in brackets (there is no comma).

If you want to make it clearer, replace by `[('component_type', '=', 'sublabel')]`, but changing the operator from `=` to `in` adds a bug here :)